io.c

```diff
@@ -9699,12 +9708,61 @@ static VALUE argf_readlines(int, VALUE *, VALUE);
 
 /*
  *  call-seq:
- *     readlines(sep=$/)     -> array
- *     readlines(limit)      -> array
- *     readlines(sep, limit) -> array
+ *    readlines(sep = $/)   -> array
+ *    readlines(limit)      -> array
+ *    readlines(sep, limit) -> array
```

Comment:
While reviewing docs and trying the examples, I discovered it also supports `**line_opts` (same as `IO#readline`: `chomp: true`). I think we need to mention it.

Reply:
I think this is new in 3.1?  I had to update to make the examples work.

Reply:
Hmm... Don't think so. Jut tried in 2.7, this works:
```
$ cat test.txt 
line1
line2
$ cat test.txt | ruby -e "p readlines"
["line1\n", "line2\n"]
$ cat test.txt | ruby -e "p readlines(chomp: true)"
["line1", "line2"]
$ ruby -v
ruby 2.7.3p183 (2021-04-05 revision 6847ee089d) [x86_64-linux]
```

Reply:
Pilot error, then.